aws/sdk-adhoc-test/build.gradle.kts

```diff
@@ -79,7 +74,7 @@ project.registerGenerateSmithyBuildTask(rootProject, pluginName, allCodegenTests
 project.registerGenerateCargoWorkspaceTask(rootProject, pluginName, allCodegenTests, workingDirUnderBuildDir)
 project.registerGenerateCargoConfigTomlTask(layout.buildDirectory.dir(workingDirUnderBuildDir).get().asFile)
 
-tasks["smithyBuildJar"].dependsOn("generateSmithyBuild")
+tasks["jar"].dependsOn("generateSmithyBuild")
```

Comment:
do we have to name the task jar? is that the normal task name? it seems a little generic, but I guess it just means "the task of making the jar?"

Reply:
So my understanding is that the `jar` task name is what comes with the Gradle `java` plugin, and now Smithy's Gradle Plugin is hooking into that existing task instead of rolling its own.